tools/sof_perf_analyzer.py

```diff
@@ -247,7 +247,10 @@ def format_perf_info() -> list[str] | None:
     '''Format SOF trace performance statistics'''
     if len(perf_stats):
         lines: list[str] = []
-        max_name_len = max(len(name) for name in component_name.values())
+        if component_name:
```

Comment:
```suggestion
        if args.kmsg:
```
Either the user provided kmsg and then it must be correct. Or they didn't and we don't have component names.

We don't want to silently ignore bad kmsg arguments.

This also saves an explanation why `component_name` can be empty.